python.json

```diff
@@ -65,5 +65,10 @@
   {"id": "py.jieba", "imports": ["jieba"], "name": "jieba", "repo": "fxsjy/jieba", "tags": ["chinese"], "tech": ["nlp"]},
   {"id": "py.django", "imports": ["django"], "name": "django", "repo": "django/django", "tags": ["framework"], "tech": ["python-web"]},
   {"id": "py.rest-framework", "imports": ["rest_framework"], "name": "rest_framework", "repo": "encode/django-rest-framework", "tags": ["django-rest"], "tech": ["python-web"]},
-  {"id": "py.matplotlib", "imports": ["matplotlib"], "name": "matplotlib", "repo": "matplotlib/matplotlib", "tags": ["visualization"], "tech": ["data-science"]}
+  {"id": "py.matplotlib", "imports": ["matplotlib"], "name": "matplotlib", "repo": "matplotlib/matplotlib", "tags": ["visualization"], "tech": ["data-science"]},
+
+  {"id": "py.django_suit", "imports": ["suit"], "name": "Django Suit", "repo": "darklow/django-suit", "tags": ["django-admin"], "tech": ["app-blocks"]},
```

Comment:
shouldn't you use '-' instead '_' in IDs ?

Reply:
right